index.html

```diff
@@ -227,7 +227,20 @@ <h3>
         </ul>
       </div>
     </section>
-
+    <section id="contact">
+      <footer>
+          <h2>Contact me</h2>
+          <p id="para">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          <form action="#" method="post">
```

Comment:
- [ ] Please make sure that the form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html).
- `action` value should be similar to `https://formspree.io/f/{form_id}`
- `method` value equals `POST`

Reply:
> * [ ]  Please make sure that the form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html).
> * `action` value should be similar to `https://formspree.io/f/{form_id}`
> * `method` value equals `POST`

- [ ] Corrected